spec/features/agents/rdvs_collectifs/full_lifecycle_spec.rb

```diff
@@ -46,22 +48,26 @@ def create_rdv_collectif(lieu_availability)
 
     click_button "Enregistrer"
     expect(page).to have_content("Atelier participatif créé")
-
     expect(page).to have_content("Jeudi 17 mars à 14:00")
-    expect(page).to have_content("3 places disponibles")
+    expect(page).to have_content("4 places disponibles")
 
     click_link("Ajouter un participant")
     add_user(user1)
 
     add_new_user
     click_button "Enregistrer"
 
-    expect(page).to have_content("1 place disponible")
+    expect(Receipt.all).to include(Receipt.where(user_id: user1.id, channel: "sms", result: "delivered").first)
```

Comment:
Ce test pourrait être écrit sous la forme `expect(Receipt.last).to have_attributes(user_id: user1.id, channel: "sms", result: "delivered")`, ce qui est un peu plus classique

Reply:
Peut-être qu'on pourrait aussi déplacer ce expect un peu plus haut pour qu'il soit au moment où le receipt est créé

Reply:
> Peut-être qu'on pourrait aussi déplacer ce expect un peu plus haut pour qu'il soit au moment où le receipt est créé

Le receipt est créé au moment ou on clique sur enregistrer ligne 58